packages/protocol/contracts/L2/TaikoL2.sol

```diff
@@ -25,8 +25,9 @@ contract TaikoL2 is EssentialContract, TaikoL2Signer, ICrossChainSync {
     using LibMath for uint256;
 
     struct Config {
+        uint128 gasExcess;
```

Comment:
if we do not use the Config object, then we can align the storage to use one less slot -- sharing the same slot with latestSyncedL1Height by changing uint128 to uint96?

Reply:
We use the config object as storage var.

Reply:
We need it because gasExcess is a constantly changing variable. not a static config var.

Reply:
Yes, i mean, get ride of the Config object, and layout the storage as:

```solidity
uint96 gasExcess; // slot 2
uint64 gasTargetPerL1Block;
uint8 basefeeAdjustmentQuotient
uint64 public latestSyncedL1Height;
```

this will only take one slot

Reply:
I'd not reduce the gasExcess size.
![kép](https://github.com/taikoxyz/taiko-mono/assets/51912515/8d0b44bf-e684-4b66-96bf-1386d395deec)